pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "djangocms-snippet"
-version = "4.1.0"
+version = "4.1.1"
```

```suggestion
dynamic = ["version"]
```

Does it make sense to make this dynamic?

You would need to add later this later in the file:

```
[tool.setuptools.dynamic]
version = {attr = "djangocms_snippet.__version__"}
```

---

We have released the version now. Can you now approve and merge this?